openpype/hosts/houdini/plugins/create/create_karma_rop.py

```diff
@@ -66,6 +66,7 @@ def create(self, subset_name, instance_data, pre_create_data):
             # we will use as render camera
             camera = None
             for node in self.selected_nodes:
+                camera = node.path()
```

Comment:
can this line be moved inside the if statement. 

as camera is updated even if it's not camera 
![image](https://github.com/ynput/OpenPype/assets/20871534/6af8e60e-c6e0-42db-9516-1232492501bd)

Reply:
it is mainly because I haven't implemented camera = node.path() into if condition. it should be fixed now.